lib/collectors.js

```diff
@@ -93,6 +97,8 @@ module.exports = (api) => {
     }
   };
 
+  api.metasync.KeyCollector.prototype = Object.create(Collector.prototype);
```

Comment:
`api.util.inherits` and fix [metasync.js](https://github.com/metarhia/MetaSync/blob/master/metasync.js) to require `util`